virt/backup_restore/virt-disaster-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-disaster-recovery"]
+= Disaster recovery
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-dr
+
+toc::[]
+
+The disaster recovery documentation provides information for administrators on
+how to recover from certain disaster scenarios that might occur with an
+{product-title} cluster. As an administrator, you must plan your {VirtProductName} deployment in advance in order to take advantage of disaster recovery.
```

Comment:
Lines should be [unwrapped](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines).